### PR TITLE
Update JG and FDB version while keeping backward compatible

### PR DESCRIPTION
**Pull request by ruweih**
Signed-off-by: Randy Hu <ruweih@gmail.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -13,8 +13,8 @@
     <properties>
         <maven.compiler.plugin.version>3.6.2</maven.compiler.plugin.version>
         <jdk.version>1.8</jdk.version>
-        <janusgraph.version>0.3.0</janusgraph.version>
-        <foundationdb.version>5.2.5</foundationdb.version>
+        <janusgraph.version>0.5.0</janusgraph.version>
```
Comment (rngcntr) [inline pom.xml:16]: You can take the latest version of JanusGrpah, which is 0.5.2

File: pom.xml [hunk 2]
```diff
@@ -13,8 +13,8 @@
...
+        <foundationdb.version>6.2.10</foundationdb.version>
```
Comment (rngcntr) [inline pom.xml:17]: Also, FDB 6.2.10 is already more than eight months old. I don't know about the state of 6.3.x, but at least 6.2.22 seems appropriate.

Comment (ruweih) [reply pom.xml]: The change has been on my laptop for a while. As suggested, both versions been updated.

**Comment by farodin91** [comment]
@ruweih I had to upgrade FDB and JG in the process of adding travis ci. 
Thanks for PR.

**Comment by farodin91** [comment]
Thanks for PR. The master use already on the current version of jg and fdb.

**Pull request closed without merging**